example_outputs.json

```diff
@@ -0,0 +1,125 @@
+# Use lists
```

Comment:
These examples would be better as test case expected outputs.  It's unclear what API is being demonstrated here.